packages/remix-node/sessions/fileStorage.ts

```diff
@@ -42,7 +42,7 @@ export function createFileSessionStorage<Data = SessionData, FlashData = Data>({
       while (true) {
         // TODO: Once node v16 is available on AWS we should use the webcrypto
         // API's crypto.getRandomValues() function here instead.
-        let randomBytes = crypto.randomBytes(8);
+        let randomBytes = crypto.webcrypto.getRandomValues(new Uint8Array(8));
```

Comment:
I don't understand why Node put Web Crypto API under `crypto.webCrypto` instead of globally like in any other runtime

Reply:
It will be under the global `crypto`, but only in Node v19
Since the new minimum version is v18, we need to do it like this unfortunately 😢

Reply:
That's nice to hear, so far I couldn't use WebCrypto in code that's not dependent of the runtime (some remix-auth strategies) because how to access the API depends if it's Node or basically anything else where it's global.

Reply:
I updated the comments in our code to point to Node v19 in #7204, so we at least know that we can use the global `crypto` once we drop support for Node 18